docs/landing/resources/api/composables.usecms.md

```diff
@@ -11,5 +11,23 @@
 <b>Signature:</b>
 
 ```typescript
-useCms: (rootContext: ApplicationVueContext) => any
+export declare function useCms(rootContext: ApplicationVueContext): {
+    page: Ref<Readonly<PageResolverProductResult | PageResolverResult<CmsPage>>>;
+    categoryId: ComputedRef<string>;
+    loading: Ref<boolean>;
+    search: (path: string, query?: any) => Promise<void>;
+    error: Ref<any>;
+    getBreadcrumbsObject: ComputedRef<PageBreadcrumb>;

```

Comment:
since we've got `useBreadcrumbs` composable and only place where we used `getBreadcrumbsObject` was `default` layout - we can mark it as deprecated because it's no longer needed I think :+1: